docs/thirdparty-openapi3-snippets.yaml

```diff
@@ -292,18 +292,9 @@ components:
             - currency
             - amount
         transactionId:
-          title: CorrelationId
-          type: string
-          pattern: '^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$'
-          description: |
-            Identifier that correlates all messages of the same sequence.
-            The API data type UUID (Universally Unique Identifier) is a JSON String
-            in canonical format, conforming to [RFC 4122](https://tools.ietf.org/html/rfc4122),
-            that is restricted by a regular expression for interoperability reasons.
-            A UUID is always 36 characters long, 32 hexadecimal symbols and 4 dashes (‘-‘).
-          example: b51ec534-ee48-4575-b6a9-ead2955b8069
+          $ref: '#/components/schemas/ConsentRequestPostRequest/properties/id'
```

Comment:
wouldn't it be better to ref to common CorrelationId

Reply:
Don't pay too much attention to the bundled file.
`swagger-cli` does this weird thing that makes schemas target other schema's parameters when there is no definition for say example `CorrelationId`

So it will import `CorrelationId` once and then every other occurances points that that import instead of the original `CorrelationId` found at the file path.

Really this file serves only 2 purposes. 
- To make sure the snippets bundle properly
- For browse-ability @ https://docs.mojaloop.io/api-snippets/

Hope this addresses your question.

Reply:
This applies to all the other comments about referecing.